src/Anthropic/Model/Claude.php

```diff
@@ -19,16 +21,28 @@ public function __construct(
     ) {
     }
 
-    public function call(MessageBag $messages, array $options = []): array
+    public function call(MessageBag $messages, array $options = []): Response
     {
         $system = $messages->getSystemMessage();
 
-        return $this->runtime->request([
+        $response = $this->runtime->request([
             'model' => $this->version->value,
             'temperature' => $this->temperature,
```

Comment:
topP should also be configurable

Reply:
oh, `$options` is ignored completely at the moment, yes